### PR TITLE
"blocker" cards

### PR DESCRIPTION
**Pull request by H1D**
- **added universal ability usage blocking logic; #160**
- **added universal turn blocking logic (skip turn); #160**
- **minor; fixed typos**
- **added underwater nouse exp. pack policy card; #160**

### CODE REVIEW

File: src/state/machines/turn.ts [hunk 1]
```diff
@@ -479,11 +480,45 @@ export const TurnMachine = setup({
...
+          outcome: "negative",
+        };
+      }),
+    ),
     unstage: assign(({ context }: { context: GameState }) =>
       produce(context, (draft) => {
         draft.stage = undefined;
       }),
     ),
+    blockAbilityUse: assign(({ context }: { context: GameState }, reasons: GamePiece["uid"]) =>
```
Comment (DreyerChris) [inline src/state/machines/turn.ts:508]: What are these actions used for?

File: src/state/machines/selectors.ts [hunk 2]
```diff
@@ -54,7 +54,10 @@ export const MachineSelectors = {
       cardBuy: getCardBoughtText(first(context.stage?.effect ?? [])),
       gameWin: i18n.t("stageEventText.gameWin"),
       gameLoss: i18n.t("stageEventText.gameLoss"),
+      abilityUseBlocked: i18n.t("stageEventText.abilityUseBlocked"),
```
Comment (DreyerChris) [inline src/state/machines/selectors.ts:57]: Are these selectors still in use?

**Review by DreyerChris — APPROVED** [review]

File: src/state/machines/turn.ts (hunk 1 above)
Comment (H1D) [reply src/state/machines/turn.ts:510]: Well... Had an idea to have kinda API thing. So we can invoke those `block/unblock` actions from exp. pack cards. Turned out it's quite hard to do. So they aren't used. I'll remove them

File: src/state/machines/selectors.ts (hunk 2 above)
Comment (DreyerChris) [reply src/state/machines/selectors.ts:57]: Oh right I understand, cool👍🏻

File: src/state/machines/turn.ts (hunk 1 above)
Comment (DreyerChris) [reply src/state/machines/turn.ts:510]: Yeah I had the same issue using external actions, one thing we can try in the future is to have a TurnMachineActionImplementations const or something, that just contains the function body logic that we can share between the main machine and the expansion "machines", but not sure if that will solve much

**Pull request merged**